tests/user_examples/test_nonisotropic_kernel/test_nonisotropic_kernel.cpp

```diff
@@ -0,0 +1,112 @@
+#include "anisotropic_kernel.hpp"
+#include "base_kernel_includes_nonisotropic.h"
+#include "kernel_wenland_c2_anisotropic.h"
+#include "sphinxsys.h"
+#include <gtest/gtest.h>
+
+using namespace SPH;
+
+Real getLinearProfile(const Vecd &input)
+{
+    return input.dot(Vecd::Ones());
+}
+
+Mat2d A{
+    {1.0, 0.0}, // First row
+    {0.0, 1.0}, // Second row
+};
+
+Real getQuadraticProfile(const Vecd &input)
+{
+    return (A * input * input.transpose()).trace();
+}
+
+TEST(test_anisotropic_kernel, test_Laplacian)
+{
+    Real PH = 1.0;                            // domain size in y direction
+    Real ratio = 4.0;                         // dp_x / dp_y
+    Real PL = ratio * PH;                     // domain size in x direction
+    int y_num = 50;                           // particle number in y direction
+    Real resolution_y = PH / Real(y_num);     // resolution in y direction
+    Real resolution_x = ratio * resolution_y; // resolution in x direction
+    int x_num = PL / resolution_x;            // Particle number in x direction , the same as particle number in y direction
+
+    Vecd scaling_vector(1.0, 1.0 / ratio); // in x and y directions
+    AnisotropicKernel<Anisotropic::KernelWendlandC2>
+        wendland(1.15 * resolution_x, scaling_vector, Vecd(0.0, 0.0)); // no rotation introduced
+
+    Mat2d transform_tensor = wendland.getCoordinateTransformationTensorG(scaling_vector, Vecd(0.0, 0.0)); // tensor
+    std::cout << transform_tensor << std::endl;
+    Mat2d tensor_D = transform_tensor * transform_tensor.transpose();
+
+    Vecd pos_i = Vecd(resolution_x * 5.0, resolution_y * 5.0); // Particle i location
```

Comment:
Vecd pos_i = Vecd(resolution_x * x_num / 2.0, resolution_y * y_num /2.0 );

Reply:
yes. you can update it laterly.